instrumentation/opentelemetry-instrumentation-django/tests/test_middleware.py

```diff
@@ -1019,6 +1020,117 @@ def tearDownClass(cls):
         super().tearDownClass()
         conf.settings = conf.LazySettings()
 
+    @staticmethod
+    def _format_request_objects_in_headers(attributes):
```

Comment:
Thanks for working on this!

Instead of redefining here in the test, could `_format_request_objects_in_headers` be imported from otel_middleware?